codecparsers/bitReader.cpp

```diff
@@ -109,9 +109,18 @@ uint32_t BitReader::read(uint32_t nbits)
     return 0;
 }
 
-void BitReader::skip(uint32_t nbits)
+bool BitReader::skip(uint32_t nbits)
 {
-    read(nbits);
+    uint32_t tmp;
+    uint32_t size = sizeof(unsigned long int);
```

Comment:
why should be tmp<<3, sizeof(unsigned long int) is the value of CACHEBYTES it is absolutely right.
Of course, i tested.